x-pack/plugins/enterprise_search/public/applications/workplace_search/app_logic.ts

```diff
@@ -27,5 +38,23 @@ export const AppLogic = kea<MakeLogicType<IAppValues, IAppActions>>({
         initializeAppData: () => true,
       },
     ],
+    isFederatedAuth: [
+      true,
+      {
+        initializeAppData: (_, { isFederatedAuth }) => !!isFederatedAuth,
+      },
+    ],
+    organization: [
+      emptyOrg,
+      {
+        initializeAppData: (_, { workplaceSearch }) => workplaceSearch?.organization || emptyOrg,
```

Comment:
I was wondering if we can tell Typescript that we know for sure that `workplaceSearch` is defined in this context. Quick googling showed Non-null assertion operator: `!.` https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-0.html#non-null-assertion-operator

I wonder if we can use it here and maybe remove the test that was added for `emptyOrg` in one of the following commits?

Reply:
Great catch Vadim! TIL the non-null assertion operator. Fixed and pushed